tests/unit/io/test_eventletreactor.py

```diff
@@ -18,50 +18,32 @@
 except ImportError:
     import unittest # noqa
 
-from tests.unit.io.utils import submit_and_wait_for_completion, TimerCallback
-from tests import is_eventlet_monkey_patched
+from tests.unit.io.utils import TimerConnectionTests
+from tests.unit.io.eventlet_utils import restore_saved_module
+from tests import notpypy
+from tests import notmonkeypatch
+
 import time
+from eventlet import monkey_patch, kill
 
 try:
     from cassandra.io.eventletreactor import EventletConnection
 except ImportError:
     EventletConnection = None  # noqa
 
-
-class EventletTimerTest(unittest.TestCase):
+@unittest.skipUnless(EventletConnection is not None, "Skpping the eventlet tests because it's not installed")
```

Comment:
Small readability nit: Could you change this to `@unittest.skipIf(EventletConnection is None, ...`? It's just a little simpler to reason about without the double negation.